src/echo.js

```diff
@@ -23,12 +27,23 @@ function _echo(opts, messages) {
   // allow strings starting with '-', see issue #20
   messages = [].slice.call(arguments, opts ? 0 : 1);
 
-  if (messages[0] === '-e') {
-    // ignore -e
+  var option = ['-e', '-n', '-ne', '-en'].indexOf(messages[0]);
+  var output;
+
+  if (option >= 0) {
```

Comment:
It might be easier to just check:

 - is the first argument a real option?
 - do the options contain `-n`?

And just switch the behavior on that condition. That way we can keep using `console.log` by default, which has some nicer behavior (see my comment below).